File: LoadingAndSaving/LoadingAndSavingExamples.vb
Imports Helpers
Imports NUnit.Framework
Imports Microsoft.VisualStudio.TestTools.UnitTesting
Imports Assert = NUnit.Framework.Assert
Imports Description = Microsoft.VisualStudio.TestTools.UnitTesting.DescriptionAttribute

Namespace Examples.LoadingAndSaving
	<TestClass, TestFixture> _
	Public Class LoadingAndSavingExamples
        <TestMethod(), Test(), Owner("WinForm")> _
	        Public Sub Excel2Word()
	            TestHelper.SetDataDir("LoadingAndSaving/Excel2Word")
	
	            Excel2WordExample.Program.Main()
        End Sub

        <TestMethod(), Test(), Owner("Console")> _
	        Public Sub CheckFormat()
	            TestHelper.SetDataDir("LoadingAndSaving/CheckFormat")
	
	            CheckFormatExample.Program.Main()
        End Sub

        <TestMethod(), Test(), Owner("Console")> _
	        Public Sub PageSplitter()
	            TestHelper.SetDataDir("LoadingAndSaving/PageSplitter")
	
	            PageSplitterExample.Program.Main()
        End Sub

	End Class
End Namespace

File: LoadingAndSaving/LoadingAndSavingExamples.cs
using Helpers;
using NUnit.Framework;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using Assert = NUnit.Framework.Assert;
using Description = Microsoft.VisualStudio.TestTools.UnitTesting.DescriptionAttribute;

namespace Examples.LoadingAndSaving
{
    [TestClass, TestFixture]
    public class LoadingAndSavingExamples
    {	
        [TestMethod, Test, Owner("WinForm")]
        public void Excel2Word()
        {
            TestHelper.SetDataDir("LoadingAndSaving/Excel2Word");

            Excel2WordExample.Program.Main();
        }

        [TestMethod, Test, Owner("Console")]
        public void CheckFormat()
        {
            TestHelper.SetDataDir("LoadingAndSaving/CheckFormat");

            CheckFormatExample.Program.Main();
        }

        [TestMethod, Test, Owner("Console")]
        public void PageSplitter()
        {
            TestHelper.SetDataDir("LoadingAndSaving/PageSplitter");

            PageSplitterExample.Program.Main();
        }

    }
}
